Fix Javadoc grammar in InstanceIdentityProvider; prefer toString() for readability in Functions

shbhmexe opened the pull request:
<!-- Comment:
Minor cleanup; no Jira needed. If backport is ever desired, a Jira could be created later.
-->

N/A

### Testing done
- No functional change. Verified equivalence of `String.valueOf(buf)` vs `buf.toString()` locally and relying on CI compilation and test suites on multiple JDKs.
- Javadoc grammar fix only.

### Proposed changelog entries
N/A

### Proposed changelog category
/label skip-changelog

### Proposed upgrade guidelines
N/A

### Submitter checklist
- [x] The changelog entries/upgrade guidelines are appropriate (skip-changelog).
- [x] Explanation provided why no tests are required (no behavior change).
- [x] New public classes/APIs: N/A
- [x] New deprecations: N/A
- [x] UI/CSP considerations: N/A
- [x] Dependency updates: N/A
- [x] New APIs with consumer links: N/A

### Desired reviewers
@jenkinsci/core-pr-reviewers

### Maintainer checklist
- [ ] There are at least two (2) approvals and no outstanding change requests.
- [x] Changelog labels are correct.
- [x] Upgrade guide label if needed (not needed here). N/A
- [x] LTS consideration only if a Jira exists (not needed here). N/A

shbhmexe committed:
Fix Javadoc grammar in InstanceIdentityProvider; use toString() for readability in Functions.toEmailSafeString

- Correct “will must be currently valid” -> “must be currently valid” in Javadoc (two occurrences)
- Prefer buf.toString() over String.valueOf(buf) for clarity; no behavior change

Signed-off-by: shbhmexe <shubhushukla586@gmail.com>

shbhmexe committed:
ci: retrigger checks

Signed-off-by: shbhmexe <shubhushukla586@gmail.com>

shbhmexe commented:
/label skip-changelog

MarkEWaite reviewed (changes requested):
Please restore the pull request template.  We rely on the pull request template to assure that the correct labels are assigned to the pull request.

MarkEWaite reviewed (approved):
Thanks!

shbhmexe committed:
Merge branch 'master' into cleanup/javadoc-instance-identity

shbhmexe committed:
tests: make ViewTest.privateView resilient to new dashboard tabs

- Detect the active view using either classic tabBar markup or the new app-build-tabs buttons.
- Avoid brittle XPath that breaks when NewDashboardPageUserExperimentalFlag is enabled.

No production code changes.

Signed-off-by: shbhmexe <shubhushukla586@gmail.com>

MarkEWaite commented:
You'll either need to revert the change that you added to this pull request in commit 69eb37eb570a24922e77c5e0a75fff74f117a022 or you'll need to update the pull request description to explain why you're making an unrelated change in the pull request.  I removed my approval of the pull request because I have not reviewed the most recent change and because the pull request description does not mention the latest change.

shbhmexe committed:
Reverted the changes introduced in the most recent commit on this branch.

Replaces support for both classic and new dashboard tab selectors with a single XPath targeting the classic tab bar. This streamlines the test by removing conditional logic for different UI layouts.

shbhmexe commented:
> You'll either need to revert the change that you added to this pull request in commit [69eb37e](https://github.com/jenkinsci/jenkins/commit/69eb37eb570a24922e77c5e0a75fff74f117a022) or you'll need to update the pull request description to explain why you're making an unrelated change in the pull request. I removed my approval of the pull request because I have not reviewed the most recent change and because the pull request description does not mention the latest change.

Hi @MarkEWaite , I’ve reverted the changes introduced in the recent commit. I had originally added those changes because some tests were failing , and I believed they would pass in a Linux environment. Since the behavior remains consistent there and no modification is actually needed, I’ve removed the changes.
If possible, could you please guide me on how to correctly fix the failing tests or point me to the relevant documentation? I’ll be happy to correct them properly. Thanks, and I hope this looks good for approval.

MarkEWaite committed:
Remove unrelated change

shbhmexe commented on test/src/test/java/hudson/model/ViewTest.java:
```diff
@@ -199,7 +199,7 @@ void privateView() throws Exception {
         HtmlPage privateViewsPage = privateViewsLink.click();
 
-        Text viewLabel = privateViewsPage.getFirstByXPath("//div[@class='tabBar']//div[@class='tab active']/a/text()");
+         Text viewLabel = privateViewsPage.getFirstByXPath("//div[@class='tabBar']//div[@class='tab active']/a/text()");
```
ok i'll do it

shbhmexe committed:
Fix indentation in ViewTest privateViewsPage section

Corrected indentation for the 'viewLabel' assignment in the ViewTest to improve code readability and maintain consistency.

shbhmexe committed:
Merge branch 'cleanup/javadoc-instance-identity' of https://github.com/shbhmexe/jenkins into cleanup/javadoc-instance-identity

shbhmexe commented:
Hi @MarkEWaite , now what should I do next? Is any change still needed or is it mergeable? Should I update the branch to make it up to date?
